driver/main.c

```diff
@@ -1053,7 +1053,12 @@ static void find_ivl_root(void)
 	/* Convert to a short name to remove any embedded spaces. */
       len = GetShortPathName(tmppath, ivl_root, sizeof ivl_root);
 #else
+	#ifdef __APPLE__
```

Comment:
Would be better to use `#if defined(__MINGW32__) ... #elif defined(__APPLE__) ... #else ... #endif` instead of a nested `ifdef`.

Reply:
Agree, updated code.